index_release.go

```diff
@@ -250,3 +253,7 @@ func validateUniqueReleases(indexReleases []IndexRelease) error {
 
 	return nil
 }
+
+func appID(a App) string {
```

Comment:
For consistency with `Authority`, it might be nice to move this to a method on `App` (`BundleID()`).

Reply:
Converted function to method, PTAL